examples/reference-entity-model/entities/Order.entity.json

```diff
@@ -1,31 +1,34 @@
-{
-  "info": {
-    "title": "Order",
-    "version": "0.0.1",
-    "baseUri": "http://marklogic.com/example/"
-  },
-  "definitions": {
-    "Order": {
-      "required": [ ],
-      "primaryKey": "orderId",
-      "properties": {
-        "orderId": {
-          "datatype": "string"
-        },
-        "address": {
-          "$ref": "#/definitions/Address"
-        }
-      }
-    },
-    "Address": {
-      "properties": {
-        "city": {
-          "datatype": "string"
-        },
-        "state": {
-          "datatype": "string"
-        }
-      }
-    }
-  }
-}
\ No newline at end of file
+{
+  "info": {
+    "title": "Order",
+    "version": "0.0.1",
+    "baseUri": "http://marklogic.com/example/"
+  },
+  "definitions": {
+    "Order": {
+      "required": [ ],
+      "primaryKey": "orderId",
+      "properties": {
+        "orderId": {
+          "datatype": "string",
+          "facet": true

```

Comment:
Let's hold off on modifying this entity - I think making birthDate and status facetable on Customer is enough for now (this is another property that a user would rarely enable a facet on).